src/test/java/com/teragrep/pth_06/planner/TableFilterTypesFromMetadataResultTest.java

```diff
@@ -0,0 +1,235 @@
+        Table<?> table = ctx
+                .meta()
+                .filterSchemas(s -> s.getName().equals("bloomdb"))
+                .filterTables(t -> !t.getName().equals("filtertype"))
+                .getTables()
+                .get(0);
+        TableFilterTypesFromMetadata result = new TableFilterTypesFromMetadata(ctx, table, 0L);
+        Result<Record> records = result.toResult();
+        Assertions.assertEquals(1, records.size());
+        Assertions.assertEquals(ULong.valueOf(1000), records.get(0).get(1));
+        Assertions.assertEquals(0.01, records.get(0).get(2));
+    }
+
+    @Test
+    void testMultipleFilterTypes() {
+        insertSizedFilterIntoTargetTable(1);
+        insertSizedFilterIntoTargetTable(2);
+        DSLContext ctx = DSL.using(conn);
+        Table<?> table = ctx
+                .meta()
+                .filterSchemas(s -> s.getName().equals("bloomdb"))
+                .filterTables(t -> !t.getName().equals("filtertype"))
+                .getTables()
+                .get(0);
+        TableFilterTypesFromMetadata result = new TableFilterTypesFromMetadata(ctx, table, 0L);
+        Result<Record> records = result.toResult();
+        Assertions.assertEquals(2, records.size());
+        Record first = records.get(0);
+        Record second = records.get(1);
+        Assertions.assertEquals(first.get(1), ULong.valueOf("1000"));
+        Assertions.assertEquals(second.get(1), ULong.valueOf("2000"));
+    }
+
+    @Test
+    public void testEquality() {
+        DSLContext ctx = DSL.using(conn);
+        Table<?> table = ctx
+                .meta()
+                .filterSchemas(s -> s.getName().equals("bloomdb"))
+                .filterTables(t -> !t.getName().equals("filtertype"))
+                .getTables()
+                .get(0);
+        TableFilterTypesFromMetadata result1 = new TableFilterTypesFromMetadata(ctx, table, 0L);
+        TableFilterTypesFromMetadata result2 = new TableFilterTypesFromMetadata(ctx, table, 0L);
+        Assertions.assertEquals(result1, result2);
+        Assertions.assertEquals(result2, result1);
+    }
+
+    @Test
+    public void testNotEquals() {
+        DSLContext ctx = DSL.using(conn);
+        Table<?> table = ctx
+                .meta()
+                .filterSchemas(s -> s.getName().equals("bloomdb"))
+                .filterTables(t -> !t.getName().equals("filtertype"))
+                .getTables()
+                .get(0);
+        TableFilterTypesFromMetadata result1 = new TableFilterTypesFromMetadata(ctx, table, 0L);
+        TableFilterTypesFromMetadata result2 = new TableFilterTypesFromMetadata(ctx, table, 1L);
+        TableFilterTypesFromMetadata result3 = new TableFilterTypesFromMetadata(ctx, null, 0L);
+        Assertions.assertNotEquals(result1, result2);
+        Assertions.assertNotEquals(result2, result1);
+        Assertions.assertNotEquals(result1, result3);
+        Assertions.assertNotEquals(result1, null);
```

Comment:
why equality tested with null?

Reply:
Cleaned up needless not equals checks against null from tests